python/tk_desktop/desktop_window.py

```diff
@@ -20,7 +20,9 @@
 
 import sgtk
 from sgtk.util import shotgun
+from sgtk.util import login
 from sgtk.platform import constants
+from tank_vendor import shotgun_authentication as sg_auth
```

Comment:
this seems to be used only in one place in the code. I suggest using the full name instead!